cli/internal/fs/path.go

```diff
@@ -148,6 +148,12 @@ func (ap AbsolutePath) RelativePathString(path string) (string, error) {
 	return filepath.Rel(ap.asString(), path)
 }
 
+// PathTo returns the relative path between two absolute paths
```

Comment:
The method name in `turbopath` land is `RelativeTo`, but no particular need to align them; I'll sort that out momentarily.